README.md

```diff
@@ -56,6 +56,7 @@ At this time, ProtectionPerms is completely based off permissions.  Without a gi
 #### Picking Up
 * Currently preventing players from picking up items is not implmented. See issue [#6](https://github.com/Zerthick/ProtectionPerms/issues/6) for more details.
 #### Dropping
+* Not Recommended. See issue [#13]
```

Comment:
You should make this a link to the actual issue like so `[#13](https://github.com/Zerthick/ProtectionPerms/issues/13)`

Reply:
I thought GitHub did that automaticly.

Reply:
It will if you don't include the square brackets I believe, if prefer explicit links though as you can still follow the link when viewed in plaintext